test/app.js

```diff
@@ -151,8 +152,15 @@ describe('generator-camel:app', function () {
     });
 
     it('Should create pom.xml with default content', function () {
-      assert.fileContent('pom.xml', new RegExp('<groupId>' + basicProps.package + '</groupId>') );
-      assert.fileContent('pom.xml', new RegExp('<artifactId>' + basicProps.name + '</artifactId>') );
+      assert.fileContent('pom.xml', new RegExp('<groupId>' + basicProps.package + '</groupId>'));
+      assert.fileContent('pom.xml', new RegExp('<artifactId>' + basicProps.name + '</artifactId>'));
+    });
+  });
+
+  describe('Should properly scaffold with default config for Java DSL', function () {
```

Comment:
I think the name of the test group is not accurate. Inside it, it is testing the validatePackage name utility methods.